packages/ndla-ui/src/RelatedArticleList/RelatedArticleList.tsx

```diff
@@ -31,6 +25,45 @@ interface RelatedArticleProps {
   type?: string;
 }
 
+const TitleWrapper = styled.div`
+  display: flex;
+  gap: ${spacing.small};
+  align-items: flex-start;
+  div {
+    min-width: 34px;
+    min-height: 34px;
+    max-width: 34px;
+    max-height: 34px;
+  }
+`;
+
+const StyledRelatedArticle = styled.article`
+  border-left: 6px solid var(--border-color, ${colors.brand.greyLight});
```

Comment:
Har du en verdi for 6?

Reply:
Mja, tja. Vi har jo spacing-verdi for det, men jeg synes det blir litt rart å bruke en spacing-verdi for noe som ikke er en spacing